code/about.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html id="top">
+
+<head>
+    <meta charset="ut-8" />
+    <meta name="viewport" content="width=device-width">
+    <title>About Us</title>
+    <link rel="stylesheet" href="./style.css" />
+</head>
+
+<header class="simple-header">
+    <nav class="nav">
+        <input type="checkbox" id="checkbox">
+        <label for="checkbox"><a class="menu-button"></a>
+        </label>
+        <div class="menu-content">
+            <a href="index.html" target="blank">home</a>
+            <a href="contacts.html" target="blank">contact</a>
+            <a href="signup.html" target="blank">sign up</a>
+            <a href="login.html" target="blank">login</a>
+        </div>
+    </nav>
+    <div class="logo"><img src="./pictures/logo_space.svg"></div>
+    <div class="logo-simple"><img src="./pictures/logo_space_simple.svg"></div>
+</header>
+
+<footer class="footer">
+    <div class="footer-wrapper">
+        <p>&copy; Pauline Andersson 2021</p>
+        <container class="footer-img-container">
```

Comment:
`<container />` tag doesn't exists, we shouldn't use it, this needs to be fixed.
Instead, consider using `<div />` tag!